src/wallet/rpcwallet.cpp

```diff
@@ -3325,6 +3325,9 @@ static UniValue bumpfee(const JSONRPCRequest& request)
         } else if (options.exists("confTarget")) { // TODO: alias this to conf_target
             coin_control.m_confirm_target = ParseConfirmTarget(options["confTarget"], pwallet->chain().estimateMaxBlocks());
         } else if (options.exists("totalFee")) {
+            if (!IsDeprecatedRPCEnabled("totalFee")) {
```

Comment:
This should call `rpcEnableDeprecated` instead of `IsDeprecatedRPCEnabled` to support wallets running in different processes from the RPC server. The aren't current examples of `rpcEnableDeprecated` in use, but you can see the last one that was removed in b4338c151d4788c33f4b7c54daaf7f94b193a624

Reply:
fixed